test/end-to-end.js

```diff
@@ -97,6 +98,20 @@ describe('end-to-end', function() {
         .expect('Access-Control-Allow-Origin',  'http://example.com')
         .expect(200, done);
     });
+    
+    it('provides security headers for all URLs ', function(done){
+      request(app).get('/')
+       .expect('X-frame-options', 'DENY')
+       .expect('x-xss-protection', '1; mode=block')
+       .expect('x-content-type-options', 'nosniff')
+       .expect('x-download-options', 'noopen')
+       .expect(function(res){
+          var headers = res.headers;
+          console.log(headers);
```

Comment:
Is this log here to work around insufficient error details printed by `headers.should.not.have.property`?

I am proposing to use `debug` instead.

```js
// at the top
var debug = require('debug')('test');

// here:
debug(headers);
```